src/main/kotlin/study/step3/car/Car.kt

```diff
@@ -0,0 +1,15 @@
+package study.step3.car
+
+class Car() {
+    var currentLocation: Int = 1
```

Comment:
```suggestion
    var currentLocation: Int = 1
        private set
```
이렇게 선언하면 외부에서 값을 변경하는 걸 막을 수 있습니다!